### DISCUSSION TITLE
Port over CTRL+A shortcut to toggle auto mod from osu!stable

### DISCUSSION THREAD

**Discussion by CenTdemeern1** [ppy/osu#17634] [Ideas] [answered]
In osu!stable, you can toggle autoplay in the song select by pressing CTRL+A on the keyboard, without opening the mod selector.
I used this hotkey a lot on osu!stable and am really missing it in lazer.
Can this be ported over, and if so, where should it be implemented (as in, which .cs file should handle the shortcut)?

**Comment by bdach** [comment]
the autoplay ctrl+enter shortcut to enter gameplay with autoplay enabled is in lazer. can you explain what is your use case for the ctrl+A one?

**Reply by CenTdemeern1** [reply]
I didn't know you could press ctrl+enter to enter a map with autoplay. The use case for ctrl+A would be "backwards compatibility" with the muscle memory of osu!stable people like me.

It's a keyboard shortcut that's very much ingrained in my mind, and probably that of others too.

**Comment by smoogipoo** [comment, marked as answer]
I don't think this one is coming back. As mentioned you can use ctrl+enter, and ctrl+a selects the search text instead.
